### PR TITLE
Adding jupyterlab extension

### PR DESCRIPTION
**Pull request by tkinz27**
Creates a jupyterlab extension with a launcher button and command to spawn the rstudio session. 

Built against Jupyterlab v0.24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,4 +29,13 @@ jupyter nbextension     install --py --sys-prefix nbrsessionproxy
 jupyter nbextension     enable  --py --sys-prefix nbrsessionproxy
 ```
 
+For JupyterLab first clone this repository to a known location and
+install from the directory.
+```
+git clone https://github.com/jupyterhub/nbserverproxy /opt/nbserverproxy
+pip install -e /opt/nbserverproxy
+juptyer serverextension enable --py nbrsessionproxy
```
Comment (ellisonbg) [inline README.md]: jupyter

File: README.md [hunk 2]
```diff
@@ -29,4 +29,13 @@ jupyter nbextension     install --py --sys-prefix nbrsessionproxy
...
+jupyter labextension link /opt/nbrsessionproxy/jupyterlab-rsessionproxy
```
Comment (ellisonbg) [inline README.md]: Probably better to publish an npm package so the user can install rather than link the package.

Comment (tkinz27) [inline README.md]: I agree, but wasn't sure if I should "own" the npm repo or let one of the original maintainers of this repository "own" the npm keys.

File: README.md (hunk 1 above)
Comment (tkinz27) [inline README.md]: Fixed... thanks

**Comment by ryanlovett** [comment]
Sorry for the delay all. I haven't tested this yet but I don't want to block for anyone either.

**Comment by ryanlovett** [comment]
And thanks for the patch! :)

**Pull request merged**